docs/get-started/tokenization/nft.md

```diff
@@ -12,7 +12,9 @@ Non-fungible tokens, or NFTs for short, are unique assets represented on the blo
 Remember that Alice wants to use blockchain to help her scale and grow her business, because it provides important properties that she cannot achieve otherwise (trust, transparency, efficiency, low costs). The first step for her is to represent her unique art pieces on-chain as NFTs. Let's go ahead and learn how to create an NFT on Algorand.
 
 # How to create NFTs
-NFTs are created using Algorand Standard Assets (ASAs), which are built into the protocol and activated using a special type of transaction. You do not need to write smart contract code, which may be the case on some other blockchains. You just need to specify a few parameters to identify it as an NFT (e.g. total count of 1) and attach metadata so that potential owners have the information they need to validate the integrity of the asset. The relevant code snippet in each of the SDKs is as follows:
+NFTs are created using Algorand Standard Assets (ASAs), which are built into the protocol and activated using a special type of transaction. You do not need to write smart contract code, which may be the case on some other blockchains. You just need to specify a few parameters to identify it as an NFT and attach metadata so that potential owners have the information they need to validate the integrity of the asset. For instance, you need to set the total amount of units you want to create for this asset to 1 and set the number of decimals to 0. This ensures you create precisely one unit of your ASA and can't divide the newly minted asset.
```

Comment:
s/activated/created

Change the sentence:
You do not need to write smart contract code, which may be the case on some other blockchains
To something more like:
This is distinct from some other blockchains where a smart contract is necessary to represent assets.

s/attach metadata/link to metadata/

The 1 is total supply true but decimals may be any number (up to protocol limit) if you're going by ARC3

Reply:
Couldn't find info on the max decimals, is it 18? Or is this defined in the algorand-go software?

Reply:
I think the max is any number up to uint32?